### PR TITLE
Feat(install): add install scripts for k8s 1.19 version

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Feat(install): add install scripts for k8s 1.19 version

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/installation/install-k8s-1.9.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    --set notary.enabled=false \
+    --set persistence.enabled=false \
+    --set trivy.ignoreUnfixed=true \
+    --set trivy.insecure=true \
+    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
+    --set core.image.tag=v2.1.0 \
+    --set harborAdminPassword="ORMBtest12345" \
+    --set storageInitializer.image=ghcr.io/kleveross/klever-ormb-storage-initializer:v0.0.7 \
+    --set predictiveUnit.defaultEnvSecretRefName=ormb \
+    --namespace harbor-system
```
Comment (simon-cj) [inline scripts/installation/install-k8s-1.9.sh:61]:
```suggestion
    helm install harbor harbor --version=v1.5.1 \
    --repo https://helm.goharbor.io \
    --set expose.nodePort.ports.http.nodePort=$HARBOR_PORT \
    --set expose.type=nodePort \
    --set expose.tls.enabled=false \
    --set trivy.enabled=false \
    --set notary.enabled=false \
    --set persistence.enabled=false \
    --set trivy.ignoreUnfixed=true \
    --set trivy.insecure=true \
    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
    --set harborAdminPassword="ORMBtest12345" \
    --namespace harbor-system
```
Test is ok in k8s v1.17.5 and v1.19.2

File: scripts/installation/install-k8s-1.9.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
```
Comment (codeflitting) [inline scripts/installation/install-k8s-1.9.sh:2]: What is the meaning of “1.9” ？

Comment (FogDong) [reply scripts/installation/install-k8s-1.9.sh:2]: the version of k8s

Comment (FogDong) [reply scripts/installation/install-k8s-1.9.sh:2]: sry it's a typo

Comment (codeflitting) [reply scripts/installation/install-k8s-1.9.sh:2]: 1.19 ？

File: scripts/installation/install-k8s-1.19.sh [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    --set global.jwtPolicy=first-party-jwt
+helm install --namespace istio-system istio-ingress istio-1.8.0/manifests/charts/gateways/istio-ingress \
+    --set global.hub="docker.io/istio" --set global.tag="1.8.0" \
+    --set global.jwtPolicy=first-party-jwt
+
+#
+# Install harbor
+#
+git clone https://github.com/kleveross/klever-model-registry.git
+kubectl create namespace harbor-system
```
Comment (simon-cj) [inline scripts/installation/install-k8s-1.19.sh:45]: why install harbor-system ns?

File: scripts/installation/install-k8s-1.19.sh [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+helm install klever-model-registry $CWD/klever-model-registry/manifests/model-registry \
+    --set externalAddress=$MASTER_IP:$KLEVER_MODEL_REGISTRY_PORT \
+    --set service.nodePort=$KLEVER_MODEL_REGISTRY_PORT \
+    --namespace=kleveross-system
+
+# Install Klever-web
+git clone https://github.com/kleveross/klever-web
+helm install klever-web $CWD/klever-web/manifests/klever-web \
+    --namespace=kleveross-system \
+    --set service.nodePort=$KLEVER_WEB_PORT
```
Comment (simon-cj) [inline scripts/installation/install-k8s-1.19.sh:96]: lost a env, please refer https://github.com/kleveross/klever-model-registry/blob/master/scripts/installation/install.sh#L101

File: scripts/installation/install-k8s-1.19.sh [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    --set expose.type=nodePort \
+    --set expose.tls.enabled=false \
+    --set trivy.enabled=false \
+    --set notary.enabled=false \
+    --set persistence.enabled=false \
+    --set trivy.ignoreUnfixed=true \
+    --set trivy.insecure=true \
+    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
+    --set harborAdminPassword="ORMBtest12345" \
+    --namespace harbor-system
```
Comment (simon-cj) [inline scripts/installation/install-k8s-1.19.sh:58]: For install harbor, we can use follow to set the helm version:
```
helm install harbor harbor --version=v1.5.1 \
    --repo https://helm.goharbor.io \
    --set expose.nodePort.ports.http.nodePort=$HARBOR_PORT \
    --set expose.type=nodePort \
    --set expose.tls.enabled=false \
    --set trivy.enabled=false \
    --set notary.enabled=false \
    --set persistence.enabled=false \
    --set trivy.ignoreUnfixed=true \
    --set trivy.insecure=true \
    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
    --set harborAdminPassword="ORMBtest12345" \
    --namespace harbor-system
```

**Comment by simon-cj** [comment]
Can run the scripts it successfully?

File: scripts/installation/install-k8s-1.19.sh (hunk 5 above)
Comment (FogDong) [reply scripts/installation/install-k8s-1.19.sh:58]: There is no differences except for the harbor chart origin.

**Comment by FogDong** [comment]
> Can run the scripts it successfully?

Test passed in k8s 1.17 & 1.19.

**Pull request merged**